### ISSUE TITLE
Unknown props `player`, `playState`, `timelineLength` on <div> tag

### ISSUE THREAD

**Issue by sarovin** [bringking/react-web-animation#39]
Hi,

i have this code:

```

<Animation
  keyframes={this.getKeyFramesLine()}
  timing={this.getTimingLine()}
  onFinish={() => this.onFinish('playText')}
>
  <div className="LineHeader" />
</Animation>
```

and in console i see this 

`warning.js:36 Warning: Unknown props`player`,`playState`,`timelineLength`on <div> tag. Remove these props from the element.`

How can i remove this warning??

**Comment by bringking** [comment]
@sarovin that must be a new warning in React. I spread those props onto the element when I was working on making it possible to animate non-native elements. I will work on a fix for that.

**Comment by bringking** [comment]
@sarovin fix published to NPM with version 0.1.1
